### ISSUE TITLE
Manuskript crashes during save process and "corrupts" the msk-file

### ISSUE THREAD

**Issue by gorgophol** [olivierkes/manuskript#352]
Hi, 

I'm using version 0.6.0 on Linux Mint 18 installed via the deb package. 
Here's what happened today in a relatively new small project (31 kb):
- Manuskript crashed after I created a scene or chapter and clicked the Metadata-Tab on the right to change the name of the scene / chapter (doesn't happen every time), Autosave worked, everything was still there after the restart and reload
- Some time later Manuskript crashed when saving a scene (800 - 1000 words), some changes were lost, but still no problem

After several crashes during the same processes as described above, I decided to switch back to 0.5.0 which ran totaly stable. I saved and closed the program after 3 hours of work, loaded again later and noticed that none of my changes have been saved since the crash. I couldn't save anything until I started again with 0.6.0 and did a correct save in a new scene. 

I tried to reproduce this and now I have the following behaviour:
- I can load but can't save (always autosave and manual save) with 0.5.0 after a crash with 0.6.0
- I can load and save with 0.6.0 (after that I can save with 0.5.0 again)
- I cannot save anything in the scene which crashed during the save process, no matter which version I use
- I cannot overwrite the msk-file using the save as function (the changes are gone, when I close and reload the program)
- I now also noticed, that Manuskript does not say, that it saved my file in those cases

Some more information:
- I can't save any changes in any chapter that was present during the crash. Only completely new scenes work
- I also tried to save as a new file. Same problem. It seems like parts of the msk file are "corrupted" somehow and just refuse any saves

I hope you may be able to reproduce this behaviour and fix it.

Cu, Ben

**Comment by gedakc** [comment]
It is possible that some metadata, such as a unique id, is missing after the failed save.  Or possible the name of the scene contains text that manuskript is unable to properly handle.

You might try the following to remove and recreate the scene which crashed on save.

1. Copy the text of the scene and paste into another editor (not manuskript).
2. Delete the scene from manuskript.
3. Create a new scene in manuskript.
4. Copy the text from the other editor and paste it into the new manuskript scene.
5. Save and exit manuskript.

Hopefully this will work around the issue you encountered.

If you discover a set of steps that reproduce the issue then please add the steps to this issue report.

**Comment by gedakc** [comment]
@gorgophol are you still experiencing this issue with crashing?
If so then please try running manuskipt from a terminal / command prompt and post the output in this report.

Have you tried the latest version of Manuskript (currently 0.7.0, with a 0.8.0 release coming soon)?

**Comment by gorgophol** [comment]
Sorry for the delay. I didn't notice your reply :-/ It works now. 

When I remember correctly, one reason I found was that I had a list of scenes in google docs, which I copied into the screen titles in Manuskript. When I copied several lines and / or had a blank or returm character at the end of the scene title in Manuskript, it caused the crash, when I saved and corrupted the file. 

I just tried again. Everything seems ok now.
